build.xml

```diff
@@ -12,14 +12,26 @@
 
   <!-- Needs to reference this in order to compile                     -->
   <!-- This path works only for OS X, Windows, and Linux must redefine -->
-  <property name="ant.jar" value="/Developer/Java/Ant/lib/ant.jar" />
+  <!-- <property name="ant.jar" value="/Developer/Java/Ant/lib/ant.jar" /> -->
 
 
   <target name="compile" description="Compile the classes">
 
     <mkdir dir="classes"/>
 
-    <javac srcdir="src" destdir="classes" source="1.6" classpath="${ant.jar}"/>
+    <!-- java 13 won't compile this with source=1.6. It moans about 1.7.
+         The first version it is silent about is 1,8, but for MacOS users
+         that has GUI problems. 9 is the first good version, so that's what
```

Comment:
That's a bit opinionated, don't you think so? I used to develop Swing GUI apps with Java 8 for years without problems...

Not sure if we should bump this to 9 per default.

Reply:
See my very long reply as to why I did all this stuff. There is a real barrier between java 8 and java 9, and I think it's worth crossing it.